cloud/aws/Makefile

```diff
@@ -0,0 +1,75 @@
+-include default.env
+
+vpc_id ?= $(eval vpc_id := $(shell bash -c 'read -p "VPC ID (existing): " input; echo $$input'))$(vpc_id)
+subnet_cidr ?= $(eval subnet_cidr := $(shell bash -c 'read -p "Subnet CIDR (to be created): " input; echo $$input'))$(subnet_cidr)
+aws_region ?= $(eval aws_region := $(shell bash -c 'read -p "AWS Region: " input; echo $$input'))$(aws_region)
+
+export AWS_REGION = ${aws_region}
+
+help:     ## Show this help.
+	@egrep -h '\s##\s' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m  %-30s\033[0m %s\n", $$1, $$2}'
+
+init:
+	terraform init
+
+docker-login:
+	aws ecr get-login-password | \
+	docker login --username AWS --password-stdin \
+		"$(shell aws sts get-caller-identity --query 'Account' --output text).dkr.ecr.${aws_region}.amazonaws.com"
+
+docker-build:
+	docker build -t tenzir/vast-lambda:latest .
```

Comment:
@dispanser / @rdettai Do we want to build this in CI and upload to Dockerhub? That should be trivial to add with a few adjustments in `.github/workflows/vast.yaml`.

Reply:
Lambda requires the image to be in a local AWS repository (ECR), which means that we need to push it there at deploy time. Having the image available in DockerHub would not be a great addition to the workflow, the only benefit I see is that instead of build+push you could do pull+push.

Adding it to the CI for building only on the other hand could be nice as it could catch some bugs if the base image is changed in a way that breaks the extra layer we are adding here. I don't really know how likely it would be that the change is big enough to break the downstream image at build time 😄

Reply:
Do you mind if I push a commit to this PR that adds it?

Reply:
did not add the docker push, as I don't think that image would be useful in docker hub, but I added the build for regression detection